### PR TITLE
Feature/#7 common carousel frontend

### PR DESCRIPTION
**Pull request by Rolly007**

### CODE REVIEW

File: frontend/src/components/CommonCarousel/CommonCarousel.jsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import Slider from 'react-slick';
+import Typography from '@material-ui/core/Typography';
+import Link from '@material-ui/core/Link';
+import ArrowForwardIosIcon from '@material-ui/icons/ArrowForwardIos';
+import ArrowBackIosIcon from '@material-ui/icons/ArrowBackIos';
+
+import MovieCard from 'components/MovieCard';
+import carouselStyles from './Carousel.module.scss';
+import Arrow from './CarouselArrow';
```
Comment (revodark) [inline frontend/src/components/CommonCarousel/CommonCarousel.jsx]: Please order your imports

File: frontend/src/components/CommonCarousel/CommonCarousel.jsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    ]
+  };
+
+  return (
+    <div className={carouselStyles.root} title='slickCarousel'>
+      <Typography variant='h5' className={carouselStyles.title}>
+        {title}
+      </Typography>
+      <Slider {...sliderSettings}>
+        {movies.map(item => {
```
Comment (revodark) [inline frontend/src/components/CommonCarousel/CommonCarousel.jsx]: I think that you could use destructuring here
({id, title, backdrop})

File: frontend/src/views/Home/Home.jsx [hunk 3]
```diff
@@ -1,9 +1,39 @@
 import React from 'react';
 
 import homeStyles from './Home.module.scss';
+import CommonCarousel from 'components/CommonCarousel';
+import GetMoviesByCategory from 'services/api/api';
+import { slice } from 'lodash';
```
Comment (revodark) [inline frontend/src/views/Home/Home.jsx]: Please order your imports

File: frontend/src/views/Home/Home.jsx [hunk 4]
```diff
@@ -1,9 +1,39 @@
...
+    { data: top, loading: loadingTop, error: errorTop }
+  ] = GetMoviesByCategory('top_rated', 'popularity.desc');
+
+  const [
+    { data: popular, loading: loadingPopular, error: errorPopular }
+  ] = GetMoviesByCategory('popular', 'popularity.desc');
+
+  const [
+    { data: now, loading: loadingNow, error: errorNow }
+  ] = GetMoviesByCategory('now_playing', 'popularity.desc');
```
Comment (revodark) [inline frontend/src/views/Home/Home.jsx]: I think that you should use the "use" prefix for the hooks

File: frontend/src/components/CommonCarousel/CommonCarousel.jsx (hunk 1 above)
Comment (Rodri77) [reply frontend/src/components/CommonCarousel/CommonCarousel.jsx]: What is your import order? can you give me a basic example please? @Rolly007 @revodark

File: frontend/src/components/CommonCarousel/CommonCarousel.jsx [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    ]
+  };
+
+  return (
+    <div className={carouselStyles.root} title='slickCarousel'>
+      <Typography variant='h5' className={carouselStyles.title}>
+        {title}
+      </Typography>
+      <Slider {...sliderSettings}>
+        {movies.map(({ id, title, backdrop }) => {
```
Comment (Rodri77) [inline frontend/src/components/CommonCarousel/CommonCarousel.jsx]: Move this logic outside render function

**Comment by Rolly007** [comment]
The commonCarousel render the following view:

commonCarousel

**Pull request merged**